lua/tree-climber.lua

```diff
@@ -281,4 +281,13 @@ M.select_node = function(options)
   vim.api.nvim_win_set_cursor(0, { end_row, end_col })
 end
 
+M.highlight_node = function(options)
+  local path, parser = get_current_node_path(options)
+  if not path or not parser then
+    return
+  end
+  local node = path[#path]
+  require 'tree-climber.highlight'.highlight_node(node, options)
```

Comment:
Can you please add the brackets here around the quotes, like in other requires? I know many people write it like that, but it always confuses me =)

Reply:
> Can you please add the brackets here around the quotes, like in other requires? I know many people write it like that, but it always confuses me =)

for this case, i think a stylua.toml could be added to this project. oh, do you use stylua or some other tools  for formatting?

Reply:
I haven't heard of stylua before so no, I do not use anything currently. Thanks for the suggestion, I will take a look at it!